docker-compose.yml

```diff
@@ -12,7 +10,16 @@ app:
   links:
     - rabbitmq
     - datomic
+    - wildfly
 rabbitmq:
   image: rabbitmq:3.5.0-management
 datomic:
   image: quay.io/democracyworks/datomic-tx:0.9.5153
+wildfly:
+  image: quay.io/democracyworks/wildfly:8.2.0.Final
+  ports:
+    - 9990
+    - 8080
+  environment:
+    ADMIN_USERNAME: admin
+    ADMIN_PASSWORD: admin
```

Comment:
The rabbitmq and datomic links will need to move to the wildfly container now (since that's where the app will actually run).